Editor/Resources/EditorWindow/Pages/GameLiftPluginBucketPopup.uss

```diff
@@ -0,0 +1,7 @@
+/*
+.custom-label {
```

Comment:
Doesn't seem like this particular style is being referenced anywhere. If that's the case could we get rid of it?